.gitignore

```diff
@@ -0,0 +1,2 @@
+*swp
+main
```

Comment:
the binary will be called `yarder`, not `main`

Reply:
Done.